pkg/virt-launcher/virtwrap/converter/network.go

```diff
@@ -52,7 +52,17 @@ func CreateDomainInterfaces(vmi *v1.VirtualMachineInstance, domain *api.Domain,
 			return nil, fmt.Errorf("failed to find network %s", iface.Name)
 		}
 
-		if iface.Binding != nil || iface.SRIOV != nil || iface.Slirp != nil {
+		var tapDomainAttachment bool
```

Comment:
Using a temp bool variable is a bit ugly.
How about defining a helper function and calling it when needed. E.g. `hasTapDomainAttachment(...)`?

Reply:
But then the same will have to be calculated twice. I don't understand how it is better or prettier.

Reply:
I think this code pattern:
```
foo := isFoo()
```

Is more readable and clear than:
```
var foo bool
if ...
   if ....
      foo = true
```

But I will not insist.

Reply:
removed the boolean.